jobs/aws/eks-distro/release-1-18-postsubmits.yaml

```diff
@@ -0,0 +1,70 @@
+# Copyright 2020 Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+postsubmits:
+  aws/eks-distro:
+  - name: release-1-18-postsubmit
+    always_run: false
+    run_if_changed: "release/1-18/RELEASE"
+    max_concurrency: 10
```

Comment:
I feel we shouldn't have more than one release job running at a time.

Reply:
+1 Agreed